src/utils/keys-utils.ts

```diff
@@ -157,16 +159,14 @@ async function getSharedSymmetricKey(
   privateKey: Uint8Array,
   ephemeralKey: Uint8Array,
 ): Promise<Optional<Uint8Array>> {
+  if ((curve25519wasm as any).init) await (curve25519wasm as any).init();
```

Comment:
We'll want to type the exports `init` and `scalarmult`, to be type safe and avoid 'any'.

(Maybe rename to `scalarMultiply` for camelcase ?)

Reply:
That's all doable :)